### PR TITLE
Fix disabling unattended upgrades 

### PR DESCRIPTION
**Pull request by olemarkus**
Current default AMIs pre-install and pre-configure unattended upgrades. We therefor need to explicitly disable it if the update policy requires it.

### CODE REVIEW

File: nodeup/pkg/model/update_service.go [hunk 1]
```diff
@@ -85,23 +85,27 @@ func (b *UpdateServiceBuilder) buildFlatcarSystemdService(c *fi.ModelBuilderCont
 }
 
 func (b *UpdateServiceBuilder) buildDebianPackage(c *fi.ModelBuilderContext) {
-	if b.NodeupConfig.UpdatePolicy != kops.UpdatePolicyExternal {
+	contents := ""
+	if b.NodeupConfig.UpdatePolicy == kops.UpdatePolicyExternal {
```
Comment (hakman) [inline nodeup/pkg/model/update_service.go:89]: External means kOps doesn't touch it not that is disabled. We need a new value for disabling explicitly.
Docs need correcting too.

Comment (olemarkus) [reply nodeup/pkg/model/update_service.go:89]: See `pkg/apis/kops/labels.go`:
```
	// UpdatePolicyExternal is a value for ClusterSpec.UpdatePolicy and InstanceGroup.UpdatePolicy indicating that upgrades are done externally, and we should disable automatic upgrades
	UpdatePolicyExternal = "external"
```

It is fairly explicit about the intended behaviour.

Comment (johngmyers) [reply nodeup/pkg/model/update_service.go:89]: `UpdateSericeBuilder.buildFlatcarSystemdService()` already uses `External` to disable explicitly.

Comment (hakman) [reply nodeup/pkg/model/update_service.go:89]: OK, seems I was wrong on this.

**Review by hakman — COMMENTED** [review]
/lgtm

**Review by hakman — APPROVED** [review]

**Pull request merged**